backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/NodeComponentProps.java

```diff
@@ -160,7 +160,7 @@ public NodeComponentProps build() {
             nodeComponentProps.cache = Objects.requireNonNull(this.cache);
             nodeComponentProps.viewCreationRequests = Objects.requireNonNull(this.viewCreationRequests);
             nodeComponentProps.viewDeletionRequests = Objects.requireNonNull(this.viewDeletionRequests);
-            nodeComponentProps.parentElementId = Objects.requireNonNull(this.parentElementId);
+            nodeComponentProps.parentElementId = this.parentElementId;
```

Comment:
No, that should stay required on purpose, see `ADR-007`

Reply:
Done